pre-commit.sh

```diff
@@ -3,9 +3,9 @@
 # Run this script before you commit.
 
 if ! [ -e venv ]; then
-  virtualenv -p python3 venv || exit 1
-  venv/bin/pip install -r requirements-server.txt || exit 1
-  venv/bin/pip install -r requirements.docs.txt || exit 1
+  virtualenv -p python3 venv || rm -rf venv; exit 1
```

Comment:
Don't you need parentheses around `rm -rf venv; exit 1`?

Reply:
I added a {} since a parenthesis would spawn a new shell so the exit wouldn't kill the script (which is the behavior I was going for).